packages/cardhost/tests/acceptance/create-card-test.js

```diff
@@ -102,19 +102,18 @@ module('Acceptance | card create', function(hooks) {
     assert.dom('.card-manipulator--right-edge--field .schema-field code').hasText("field: title");
   });
 
-  // TODO: un-skip when we add multiple drop zones
-  skip('can add a field at a particular position', async function(assert) {
+  test('can add a field at a particular position', async function(assert) {
     await login();
     await visit('/cards/new');
 
     assert.equal(currentURL(), '/cards/new');
 
     await setCardId(card1Id);
     await addField('title', 'string', true);
-    await addField('body', 'string', false);
+    await addField('body', 'string', false, 1);
     await addField('author', 'string', false, 1);
 
-    assert.deepEqual([...document.querySelectorAll('[data-test-field]')].map(i => i.getAttribute('data-test-field')),
+    assert.deepEqual([...document.querySelectorAll('[data-test-isolated-card="local-hub::article-card::millenial-puppies"] [data-test-field]')].map(i => i.getAttribute('data-test-field')),
```

Comment:
I made a const for the magic constant:
```js
const card1Id = 'local-hub::article-card::millenial-puppies';
```
you can just use that instead of having to type this all out.

Reply:
in my defense I didn't actually type it in :)